testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-oss/spark-defaults.conf

```diff
@@ -1,3 +1,5 @@
+spark.driver.memory=2g
```

Comment:
Any idea why we need this here and not in 8 other spark-defaults.conf files?
let's capture the reasoning in a code comment

Reply:
Added a code comment. There're some differences (Spark version, number of product tests and so on), but I'm not sure about the root cause, to be honest.